glusterfsd/src/glusterfsd.c

```diff
@@ -1377,6 +1379,9 @@ parse_opts(int key, char *arg, struct argp_state *state)
                              arg);
             }
 
+            break;
+        case ARGP_FUSE_FSNAME_KEY:
+            cmd_args->fs_display_name = gf_strdup(arg);
```

Comment:
Where is this memory released ?

Reply:
Was not able to find a proper place where these things are freed from cmd_args. But realized that most of them get set with dictionary for fuse options, which frees the data. Doing the same now for this.